core/jsAndWasmShared/src/kotlinx.datetime.internal.JSJoda.module_@js-joda_core.kt

```diff
@@ -68,7 +68,6 @@ external open class ValueRange : InteropInterface {
 external open class TemporalAmount : InteropInterface {
     open fun <T : Temporal> addTo(temporal: T): T
     open fun get(unit: TemporalUnit): Int
-//    open fun units(): JsArray<TemporalUnit>
```

Comment:
I removed some methods from datetime js public API. This API is kotlin layer over jsJoda. Is this API needed to preserve?